client/sync.js

```diff
@@ -96,6 +96,75 @@ const startSync = (requester) => {
     return jsRecords
   }
 
+  const getBaseBookmarksOrder = (deviceId, platform) => {
+    let platformPrefix = `1.`
+    if (platform === `ios` || platform === `android`) {
+      platformPrefix = `2.`
+    }
+    platformPrefix += deviceId + `.`
+    return platformPrefix
+  }
+
+  const getBookmarkOrder = (prevOrder, nextOrder) => {
```

Comment:
this should handle the case where prevOrder or nextOrder are null/empty (in order to get a bookmark at the start of the list or at the end of the list) - see comment in https://docs.google.com/a/brave.com/document/d/1t2IEFQF0Xj2aNnlEf0KaIXEqbfQ_RCn4b01LuIuef6I/edit?disco=AAAABxDSFGk

Reply:
^ nvm, i see you assume they are empty strings instead of null. 👍